data/sql/updates/pending_db_world/rev_1592179496642830400.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1592179496642830400');
+
+CREATE TABLE IF NOT EXISTS `world_config` (
+  `name` VARCHAR(255) NOT NULL,
+  `value` INT UNSIGNED NOT NULL,
+  `comment` TEXT,
+   PRIMARY KEY (`name`)
+) ENGINE=MyISAM DEFAULT CHARSET=UTF8MB3 COMMENT='Holds configs for the World Server';
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS `world_config` (
  `type` TINYINT UNSIGNED NOT NULL,
  `name` VARCHAR(255) NOT NULL,
  `value` INT UNSIGNED NOT NULL,
  `comment` TEXT,
   PRIMARY KEY (`type`, `name`)
) ENGINE=MyISAM DEFAULT CHARSET=UTF8MB3 COMMENT='Holds configs for the World Server';
```

Actually, what about adding a `type` column too?
That way we could sort and store the configs by type, for example
```
1 - Server Settings
2 - Warden Settings
3 - Player Interactions
100 - Ungrouped
```
So on so forth.
I think that could be a good idea, so that we don't just have a table full of random configs just laying around